e2e/test_options.go

```diff
@@ -25,12 +25,26 @@ import (
 	"github.com/mysterium/node/tequilapi/client"
 )
 
-var tequilaHost = flag.String("tequila.host", "localhost", "Specify tequila host for e2e tests")
+type Domain int
+
+const (
+	Client Domain = 0
```

Comment:
Do we really need a one function with ifs or switches inside. What about two separate functions:
1. NewClientTequilAPI(...)
2. NewNodeTequilAPI(...)